explainers/beforematch.md

```diff
@@ -288,6 +307,50 @@ scroll after adding an async step.
 5. **content-visibility: hidden-matchable -> visible**: find-in-page scrolls to
 to the revealed text.
 
+### Alternatives
+Given the purpose of displaying `content-visibility: hidden-matchable` text
+when it is searched for, there are a number of alternatives we have considered.
+
+#### Automatic Unlocking
+`content-visibility: hidden-matchable` text would be automatically changed to
+`content-visibility: visible` when the hidden text is searched for.
+##### Pros
+* Provides the desired behavior without the need for the web developer to do
+  anything besides use the `hidden-matchable` value.
+* Less complex than `beforematch` with regards to scrolling behavior.
+##### Cons
+* Doesn't allow the developer to change other properties in conjunction with
+  displaying hidden content. For example, in "Example 1: Expanding
+  `hidden-matchable`," automatic unlocking wouldn't be able to change the
+  style of the `<h1>` title elements.
+
+#### Automatic Unlocking with Activation Event
+This is like "Automatic Unlocking," but with an added "activation" emitted when
+content is displayed to allow developers to change other state and styles if
+needed.
+##### Pros
+* Same pros as "Automatic Unlocking."
+* Allows other state/styles to be changed when content is displayed.
+##### Cons
+* TODO: ask vmpstr and chrishtr for more context on this
+
+#### CSS Pseudo Selector
+A pseudo selector, such as `:target`, would be applied to the element
+containing the matching text when it is searched for.
+##### Pros
+* Allows content to become visible when searched for with only CSS.
+##### Cons
+* If CSS with a pseudo selector is used to make text visible, then when
```

Comment:
Is it possible for you to take a few screenshots on a phone interaction with mobile wiki page
- One screen of collapsed sections
- One screen of find-in-page that finds text in a collapsed section (you can manually expand the section for the purpose of the demo)
- One screen with find-in-page dialog dismissed that still displays the open section.

I think that example would make this con point obvious

Reply:
Nevermind, I see you added a similar thing in the #172